test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AuthorizationApplicationModelProviderTest.cs

```diff
@@ -57,8 +57,8 @@ public void BuildActionModels_BaseAuthorizeFiltersAreStillValidWhenOverriden()
             var authorizeFilters = action.Filters.OfType<AuthorizeFilter>();
             Assert.Single(authorizeFilters);
 
-            Assert.NotNull(authorizeFilters.First().PolicyProvider);
-            Assert.Equal(2, authorizeFilters.First().AuthorizeData.Count()); // Base + Derived authorize
+            Assert.NotNull(authorizeFilters.First().Policy);
+            Assert.Equal(3, authorizeFilters.First().Policy.Requirements.Count()); // Basic + Basic2 + Derived authorize
```

Comment:
This seems like it would be equivalent, but the fact that we have an extra requirement bothers me. Will look into it more and consult an Auth expert tomorrow.